tests/components/ring/test_sensor.py

```diff
@@ -48,3 +52,26 @@ async def test_sensor(hass: HomeAssistant, requests_mock: requests_mock.Mocker)
     )
     assert front_door_wifi_signal_strength_state is not None
     assert front_door_wifi_signal_strength_state.state == "-58"
+
+
+async def test_only_chime_devices(
+    hass: HomeAssistant, requests_mock: requests_mock.Mocker, caplog
+) -> None:
+    """Tests the update service works correctly if only chimes are returned."""
+    requests_mock.get(
+        "https://api.ring.com/clients_api/ring_devices",
+        text=load_fixture("chime_devices.json", "ring"),
+    )
```

Comment:
We usually patch the library instead of mocking http requests done by the integration

Reply:
All of the tests in the integration use this approach for mocking the library so it could be a quite lift to change that for this fix .  I can look at seeing how to change this for all the tests in a subsequent PR if that's ok.